tikvrpc/interceptor/interceptor_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 TiKV Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package interceptor
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/tikv/client-go/v2/tikvrpc"
+)
+
+func TestInterceptor(t *testing.T) {
+	chain := NewRPCInterceptorChain()
+	manager := MockInterceptorManager{}
+	it := chain.
```

Comment:
I suggest we should test the order in which the linked interceptors are called.

Reply:
updated at [L28-L39](https://github.com/tikv/client-go/pull/389/files#diff-6d91bc07e1d85d86f29b8599bd4e0c023a5033ef01b6e1b981cbaeb5579681baR28-R39)